### DISCUSSION TITLE
Tag that makes enemies immune to all cactus-like blocks and decorators like Quartz Spikes in the Quartz Desert, Warped Cactus, etc.

### DISCUSSION THREAD

**Discussion by Neko2n** [CorgiTaco/BYG#1184] [Ideas]
**Is your feature request related to a problem? Please describe.**
Everything in these biomes just constantly dies, and the death sounds are annoying. Not only that, but when I try to make enemies spawn in these biomes they die too so it's basically impossible to make these biomes challenging outside of the terrain.

**Describe the solution you'd like**
Having a datapackable entity tag that makes mobs immune to cactus damage and spike damage from these sources would solve this immediately, plus allowing compat with any modded entity via. datapacking.

**Describe alternatives you've considered**
Can't really come up with any honestly. An immunity tag is the only solution I've thought of, there's not much of a way to fix this myself right now.

**Additional context**
_Pretend there's a screenshot here of skeletons getting killed by quartz spikes_

**Comment by Darkmega18** [comment]
I can probably get behind this too. make sure you include the sweet berries also.
